test/Swashbuckle.AspNetCore.SwaggerGen.Test/XmlComments/XmlCommentsTextHelperTests.cs

```diff
@@ -158,5 +158,36 @@ This is a paragraph.
 
             Assert.Equal("\r\nThis is a paragraph. MultiLined.\r\n\r\nThis is a paragraph.", output, false, true);
         }
+
+        [Fact]
+        public void Humanize_CodeMultiLineTag()
+        {
+            const string input = @"
+            <code>
+               {
+                ""Prop1"":1,
+                ""Prop2"":[]
+               }
+            </code>";
+
+            var output = XmlCommentsTextHelper.Humanize(input);
+
+            Assert.Equal("```\r\n   {\r\n    \"Prop1\":1,\r\n    \"Prop2\":[]\r\n   }\r\n```", output, false, true);
```

Comment:
Could you factor these expected values into something a bit easier to read? With the expected additional indentation it's harder to parse than the others in this file.

Maybe something like this?

```csharp
var expected = string.Join("\r\n",
[
    "   {",
    "    \"Prop1\":1,",
    "    \"Prop2\":[]",
    "   }"
]);

Assert.Equal(expected, output, false, true);
```

Reply:
good idea, applied suggestion